search-app/docs/api.md

```diff
@@ -1695,6 +1697,12 @@ Lines are provided as a list of comma separated values representing coordinates
 
     curl "%CMR-ENDPOINT%/collections?line=-0.37,-14.07,4.75,1.27,25.13,-15.51"
 
+##### <a name="c-circle"></a> Circle
```

Comment:
Wonder if `radius` would be more clear than `circle`. thats super nit-picky though

Reply:
If we use `radius`, where is the `point` info? I am not sure I follow.

Reply:
Fair enough